parsing/lexer.mll

```diff
@@ -413,7 +413,11 @@ rule token = parse
         let s, loc = with_comment_buffer comment lexbuf in
         COMMENT (s, loc) }
   | "(*" ('*'*) as stars "*)"
-      { COMMENT (stars, Location.curr lexbuf) }
+      { if !handle_docstrings && stars="(*" then
```

Comment:
I also thought so at the start, but the "stars" matches the whole `"(*" ('*'*)` sub-expression and not only the `('*'*)` part.  Ading parenthesis, `"(*" (('*'*) as stars)`, fixes this problem. Should I add a commit to fix this issue?